main.js

```diff
@@ -0,0 +1,28 @@
+const viewportWidth = window.innerWidth;
+const header = document.querySelector('#header');
+const burger = document.querySelector('#burger');
+
+let showMenu = false;
+
+function toggleMenu() {
+  if (!showMenu && viewportWidth < 768) {
+    header.classList.remove('header');
+    header.classList.add('header-active');
+    burger.classList.remove('burger');
+    burger.classList.add('burger-active');
+    document.body.style.overflowY = 'hidden';
+
+    // set  menu state
+    showMenu = true;
+  } else {
+    header.classList.remove('header-active');
+    header.classList.add('header');
+    burger.classList.remove('burger-active');
+    burger.classList.add('burger');
+    document.body.style.overflowY = 'auto';
+
+    // Set menu state
```

Comment:
hello there currently when i click on the hamburger menu it doesn't behave as expected. it currently doesn't work at all please fix this little bug and try again 😉